openedx/features/caliper_tracking/tests/tests.py

```diff
@@ -1,11 +1,12 @@
 """
 This module contains the test cases for caliper_tracking application
 """
-
+import mock
 import json
 import os
 from datetime import datetime
 
+from openedx.features.caliper_tracking import utils
```

Comment:
@tehreem-sadat  Is this really important?

Reply:
Removed.